src/jest.mli

```diff
@@ -110,6 +110,8 @@ module Expect : sig
   val toEqual : 'a -> [< 'a partial] -> assertion
   val toHaveLength : int -> [< 'a array partial] -> assertion
   val toMatch : string -> [< string partial] -> assertion
+  val toMatchInlineSnapshot : string -> _ plainPartial -> assertion
+  val toMatchSnapshot : _ plainPartial -> assertion
   val toMatchRe : Js.Re.t -> [< string partial] -> assertion
   val toMatchSnapshot : _ plainPartial -> assertion
```

Comment:
Seems like `toMatchSnapshot` has been accidentally duplicated. I would expect this to have produced some kind of warning at least though.

```suggestion
  val toMatchInlineSnapshot : string -> _ plainPartial -> assertion
  val toMatchRe : Js.Re.t -> [< string partial] -> assertion
  val toMatchSnapshot : _ plainPartial -> assertion
```

Reply:
oups, goog catch. I would have expected the same. But I'm just starting with ocaml so I wouldn't know better.